llarp/service/endpoint.cpp

```diff
@@ -75,10 +75,10 @@ namespace llarp
     Endpoint::RegenAndPublishIntroSet(bool forceRebuild)
     {
       const auto now = llarp::time_now_ms();
-      std::set< Introduction > I;
+      std::set< Introduction > introset;
       if(!GetCurrentIntroductionsWithFilter(
-             I, [now](const service::Introduction& intro) -> bool {
-               return not intro.ExpiresSoon(now, 2 * 60 * 1000);
+             introset, [now](const service::Introduction& intro) -> bool {
+               return not intro.ExpiresSoon(now, 10 * 60 * 1000);
```

Comment:
this can be `std::chrono::milliseconds(path::default_lifetime) / 2` instead of changing the path lifetime

Reply:
I'll make this a constant value (grouped with other similar ones) once this is rebased on top of the `chrono` PR. Similarly with the `buildSpread` variable.